xbmc/interfaces/json-rpc/PlayerOperations.cpp

```diff
@@ -1104,6 +1104,29 @@ JSONRPC_STATUS CPlayerOperations::SetAudioStream(const std::string &method, ITra
   return ACK;
 }
 
+JSONRPC_STATUS CPlayerOperations::AddSubtitle(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
+{
+  switch (GetPlayer(parameterObject["playerid"]))
+  {
+    case Video:
+        break;
```

Comment:
This looks odd. Either just use `if (GetPlayer(parameterObject["playerid"]) != Video) return FailedToExecute;` or move the actual implementation into the `switch-case` as done in `SetAudioStream()` et al.